presto-docs/src/main/sphinx/functions/ip.rst

```diff
@@ -60,3 +60,13 @@ IP Functions
         SELECT IP_PREFIX_COLLAPSE(ARRAY[IPPREFIX '2620:10d:c090::/48', IPPREFIX '2620:10d:c091::/48']); -- [{2620:10d:c090::/47}]
         SELECT IP_PREFIX_COLLAPSE(ARRAY[IPPREFIX '192.168.1.0/24', IPPREFIX '192.168.0.0/24', IPPREFIX '192.168.2.0/24', IPPREFIX '192.168.9.0/24']); -- [{192.168.0.0/23}, {192.168.2.0/24}, {192.168.9.0/24}]
 
+.. function:: is_private_ip(ip_address) -> boolean
+
+    Returns whether ``ip_address`` of type ``IPADDRESS`` is a private or reserved IP address
+    that is not considered globally reachable by IANA. `Null` inputs return `null`. ::
```

Comment:
Consider adding the links added in [IpPrefixFunctions.java](https://github.com/prestodb/presto/pull/23520/files#diff-4c47f717e68d7849cc1528843f323f9fcd0801c2a01e41db20fcb914b0f11cda) to the documentation. 

Maybe something like the following: 

```suggestion
    that is not considered globally reachable by IANA. For more information, see [IANA IPv4 Special-Purpose Address Registry](https://www.iana.org/assignments/iana-ipv4-special-registry/iana-ipv4-special-registry.xhtml) and [IANA IPv6 Special-Purpose Address Registry](https://www.iana.org/assignments/iana-ipv6-special-registry/iana-ipv6-special-registry.xhtml). `Null` inputs return `null`. ::
```